code/modules/jobs/job_types/cargo_service.dm

```diff
@@ -152,10 +152,11 @@ Cook
 /datum/outfit/job/cook/pre_equip(mob/living/carbon/human/H)
 	..()
 	var/datum/job/cook/J = SSjob.GetJob(H.job)
-	J.cooks++
-	if(J.cooks>1)//Cooks
-		suit = /obj/item/clothing/suit/apron/chef
-		head = /obj/item/clothing/head/soft/mime
+	if(J)
+		J.cooks++
+		if(J.cooks>1)//Cooks
+			suit = /obj/item/clothing/suit/apron/chef
+			head = /obj/item/clothing/head/soft/mime
```

Comment:
it doesn't really fix the underlying issue, GetJob() should return the correct cook job, no?